app/controllers/alaveteli_pro/stripe_webhooks_controller.rb

```diff
@@ -18,6 +18,11 @@ def receive
       end
     rescue UnhandledStripeWebhookError => e
       notify_exception(e)
+    rescue NoMethodError => e
```

Comment:
This rescue will also pickup any `NoMethodError` exceptions raise within a matching case/when statement - not just for when `@stripe_event.type` doesn't exist. I think this could be confusing, do we want to be more explicit here?